src/http/mod.rs

```diff
@@ -51,17 +49,14 @@ use std::{
 };
 
 lazy_static! {
-    static ref CLIENT: HyperClient = {
-        let tc = NativeTlsClient::new().expect("Unable to make http client");
-        let connector = HttpsConnector::new(tc);
-
-        HyperClient::with_connector(connector)
+    static ref CLIENT: ReqwestClient = {
```

Comment:
Style nitpick: This is just a simple call to `ReqwestClient::new()` now, the braces the can be removed.
```suggestion
    static ref CLIENT: ReqwestClient = ReqwestClient::new();
```